packages/fxa-settings/src/pages/CookiesDisabled/en.ftl

```diff
@@ -0,0 +1,4 @@
+cookies-disabled-header = Cookies and local storage are required
+cookies-disabled-enable-prompt = Please enable cookies and local storage in your browser to access { -product-firefox-accounts }. Doing so will enable functionality such as remembering you between sessions.
+# Button text. Users can click to check if cookies and local storage are enabled and be directed to the previous page if so.
+cookies-disabled-try-again = Try again
```

Comment:
maybe more a question for @bcolsson, but does this need a "-btn" suffix or something to indicate usage (although we do mention "Button text" in the description)?
I did a quick grep (big shock) but not sure how consistently this is applied:

```sh
git grep -n "\-btn" packages/**/*.ftl | cat

packages/fxa-payments-server/src/components/PaymentForm/en.ftl:9:payment-cancel-btn = Cancel
packages/fxa-payments-server/src/components/PaymentForm/en.ftl:10:payment-update-btn = Update
packages/fxa-payments-server/src/components/PaymentForm/en.ftl:11:payment-pay-btn = Pay now
packages/fxa-payments-server/src/components/PaymentForm/en.ftl:12:payment-pay-with-paypal-btn = Pay with {-brand-name-paypal}
packages/fxa-payments-server/src/routes/Subscriptions/en.ftl:28:pay-update-change-btn = Change
packages/fxa-payments-server/src/routes/Subscriptions/en.ftl:29:pay-update-manage-btn = Manage
```

Reply:
So when it comes to localization, clearer is always better.

The fact it is a button being called out in the comment is great. Having it in the ID is one more point of reference that could be caught by the localizer. Some localizers may also have alternative workflows than working directly in Pontoon, such as using external CAT tools which may lose the comment context. Having `-button` in the identifier would preserve this in most contexts I can think of.

My preference would be to have something in the ID as well (though not enough to block the PR). 

If we are going to add it here `-button` would be preferable to `-btn` as abbreviations may not be as easily understood by non-native English speakers.

```bash
git grep -n "\-button" packages/**/*.ftl | cat
```
```
packages/fxa-auth-server/lib/senders/emails/templates/recovery/en.ftl:6:recovery-new-password-button = Create a new password by clicking the button below. This link will expire within the next hour.
packages/fxa-payments-server/src/components/PaymentConfirmation/en.ftl:44:payment-confirmation-download-button = Continue to download
packages/fxa-payments-server/src/components/PaymentErrorView/en.ftl:3:payment-error-retry-button = Try again
packages/fxa-payments-server/src/components/PaymentErrorView/en.ftl:4:payment-error-manage-subscription-button = Manage my subscription
packages/fxa-payments-server/src/components/PaymentErrorView/en.ftl:12:iap-upgrade-get-help-button = Get help
packages/fxa-payments-server/src/components/PlanDetails/en.ftl:5:plan-details-show-button = Show details
packages/fxa-payments-server/src/components/PlanDetails/en.ftl:6:plan-details-hide-button = Hide details
packages/fxa-payments-server/src/routes/Subscriptions/Reactivate/en.ftl:19:reactivate-confirm-button = Resubscribe
packages/fxa-payments-server/src/routes/Subscriptions/Reactivate/en.ftl:25:reactivate-success-button = Close
packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionIapItem/en.ftl:5:sub-iap-item-manage-button = Manage
packages/fxa-settings/src/components/FormResetPasswordWithBalloon/en.ftl:7:form-reset-password-with-balloon-submit-button = Reset password
packages/fxa-settings/src/components/Settings/ConnectedServices/en.ftl:14:cs-refresh-button =
packages/fxa-settings/src/components/Settings/ConnectedServices/en.ftl:56:cs-sign-out-button = Sign out
packages/fxa-settings/src/components/Settings/LinkedAccounts/en.ftl:5:la-unlink-button = Unlink
packages/fxa-settings/src/components/Settings/LinkedAccounts/en.ftl:6:la-unlink-account-button = Unlink
packages/fxa-settings/src/components/Settings/Modal/en.ftl:4:modal-cancel-button = Cancel
packages/fxa-settings/src/components/Settings/ModalVerifySession/en.ftl:9:msv-cancel-button = Cancel
packages/fxa-settings/src/components/Settings/ModalVerifySession/en.ftl:10:msv-submit-button-2 = Confirm
packages/fxa-settings/src/components/Settings/PageAvatar/en.ftl:6:avatar-page-add-photo-button =
packages/fxa-settings/src/components/Settings/PageAvatar/en.ftl:9:avatar-page-take-photo-button =
packages/fxa-settings/src/components/Settings/PageAvatar/en.ftl:12:avatar-page-remove-photo-button =
packages/fxa-settings/src/components/Settings/PageAvatar/en.ftl:15:avatar-page-cancel-button = Cancel
packages/fxa-settings/src/components/Settings/PageAvatar/en.ftl:16:avatar-page-save-button = Save
packages/fxa-settings/src/components/Settings/PageAvatar/en.ftl:17:avatar-page-saving-button = Saving…
packages/fxa-settings/src/components/Settings/PageAvatar/en.ftl:18:avatar-page-zoom-out-button =
packages/fxa-settings/src/components/Settings/PageAvatar/en.ftl:20:avatar-page-zoom-in-button =
packages/fxa-settings/src/components/Settings/PageAvatar/en.ftl:22:avatar-page-rotate-button =
packages/fxa-settings/src/components/Settings/PageChangePassword/en.ftl:12:pw-change-cancel-button = Cancel
packages/fxa-settings/src/components/Settings/PageChangePassword/en.ftl:13:pw-change-save-button = Save
packages/fxa-settings/src/components/Settings/PageDeleteAccount/en.ftl:33:delete-account-continue-button = Continue
packages/fxa-settings/src/components/Settings/PageDeleteAccount/en.ftl:38:delete-account-cancel-button = Cancel
packages/fxa-settings/src/components/Settings/PageDeleteAccount/en.ftl:39:delete-account-delete-button-2 = Delete
packages/fxa-settings/src/components/Settings/PageRecoveryKeyAdd/en.ftl:3:recovery-key-cancel-button = Cancel
packages/fxa-settings/src/components/Settings/PageRecoveryKeyAdd/en.ftl:4:recovery-key-close-button = Close
packages/fxa-settings/src/components/Settings/PageRecoveryKeyAdd/en.ftl:5:recovery-key-continue-button = Continue
packages/fxa-settings/src/components/Settings/PageSecondaryEmailAdd/en.ftl:9:add-secondary-email-cancel-button = Cancel
packages/fxa-settings/src/components/Settings/PageSecondaryEmailAdd/en.ftl:10:add-secondary-email-save-button = Save
packages/fxa-settings/src/components/Settings/PageSecondaryEmailVerify/en.ftl:9:verify-secondary-email-cancel-button = Cancel
packages/fxa-settings/src/components/Settings/PageSecondaryEmailVerify/en.ftl:10:verify-secondary-email-verify-button-2 = Confirm
packages/fxa-settings/src/components/Settings/PageTwoStepAuthentication/en.ftl:9:tfa-button-continue = Continue
packages/fxa-settings/src/components/Settings/PageTwoStepAuthentication/en.ftl:10:tfa-button-cancel = Cancel
packages/fxa-settings/src/components/Settings/PageTwoStepAuthentication/en.ftl:11:tfa-button-finish = Finish
packages/fxa-settings/src/components/Settings/PageTwoStepAuthentication/en.ftl:29:tfa-button-cant-scan-qr = Can’t scan code?
packages/fxa-settings/src/components/Settings/UnitRowTwoStepAuth/en.ftl:10:tfa-row-button-refresh =
packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/en.ftl:18:account-recovery-confirm-key-button = Confirm account recovery key
packages/fxa-settings/src/pages/ResetPassword/ResetPassword/en.ftl:13:reset-password-button = Begin reset
packages/fxa-settings/src/pages/Signin/SigninRecoveryCode/en.ftl:19:signin-recovery-code-confirm-button = Confirm
packages/fxa-settings/src/pages/Signin/SigninTokenCode/en.ftl:14:signin-token-code-confirm-button = Confirm
packages/fxa-settings/src/pages/Signin/SigninTotpCode/en.ftl:19:signin-totp-code-confirm-button = Confirm
packages/fxa-settings/src/pages/Signin/en.ftl:14:signin-button = Sign in
```

Reply:
Great, now I have to live with knowing we have `-btn` and `-button` suffixes (although the `-btn` definitely seems like the outlier &mdash; and I'm not convinced it's a big enough deal to change the 6 instances [in each locale]).